### PR TITLE
Beer song instance methods

### PR DESCRIPTION
**Pull request by vasouv**
Oops, because I moved a couple of curly brackets, the diff is weird now. Should I fix in case it's confusing?

Also, question: why do we have ignore annotations if gradle runs all the tests anyway?

### CODE REVIEW

**Comment by FridaTveit** [comment]
That's fine, I'm glad you moved the curly brackets, it looks better now :) Doing `gradle build` runs all the tests but doing `gradle test` (which is what we tell the people doing the exercises to do) only runs the ones without `@Ignore` :)

**Comment by vasouv** [comment]
You're absolutely right! Now that you've mentioned it, I recall reading in the docs that the build runs all tests for devs, and ignores some of them for the users. I get it.

Thanks for the clarification :)

File: exercises/beer-song/src/test/java/BeerSongTest.java [hunk 1]
```diff
@@ -9,39 +9,39 @@
     public void singFirstVerse() {
         assertEquals("99 bottles of beer on the wall, 99 bottles of beer.\n" +
                         "Take one down and pass it around, 98 bottles of beer on the wall.\n\n",
-                BeerSong.verse(99));
+                new BeerSong().verse(99));
```
Comment (FridaTveit) [inline exercises/beer-song/src/test/java/BeerSongTest.java]: If you make a private BeerSong variable, then you can initialise it in a setup method and just do `beerSong.verse(99)` here :) That way you don't have to duplicate the initialisation in every test :)

**Comment by vasouv** [comment]
Fixed. And to think my previous pull request was doing exactly that at the wordy exercise :P

**Pull request merged**